splink/linker.py

```diff
@@ -378,6 +378,43 @@ def _execute_sql_against_backend(self, sql, templated_name, physical_name):
             f"_execute_sql_against_backend not implemented for {type(self)}"
         )
 
+    def register_table(self, input, table_name):
+        """
```

Comment:
Need to add this to the docs [here](https://github.com/moj-analytical-services/splink/blob/65586d05a6523ae1f6535d85d79fa65cef75ab0a/docs/linker.md#L40)